glossary.yml

```diff
@@ -1091,11 +1161,39 @@
     def: >
       A vote against something.
 
+- slug: dynamic_loading
+  en:
+    term: "dynamic loading"
+    def: >
+      To [import](#import) a [module](#module) into the memory of a program
+      while it is already running. Most [interpreted languages](#interpreted_language)
+      use dynamic loading, and provide tools so that programs can find and load
+      modules dynamically to configure themselves
+
+- slug: dynamic_lookup
+  en:
+    term: "dynamic lookup"
+    def: >
+      To find a function or a property of an [object](#object) by name while a
+      program is running. For example, instead of getting a specific property
+      of an object using `obj.name`, a program might use `obj[someVariable]`,
+      where `someVariable` could hold `"name"` or some other property name.
+
+- slug: dynamic_scoping
+  en:
+    term: "dynamic scoping"
+    def: >
+      To find the value of a [variable](#variable_program) by looking at what is
+      on the [call stack](#call_stack) at the moment the lookup is done. Almost
+      all programming languages use [lexical_scoping](#lexical_scoping) instead,
+      since it is more predictable.
+
 - slug: emacs
   en:
     term: "Emacs (editor)"
     def: >
-      FIXME
+      A [Lisp](#lisp)-based operating system disguised as a text editor that is
```

Comment:
I'm reading this as a bit of an in-joke among Emacs users that may not necessarily translate for a more general user. Can you rephrase this?

Reply:
Yeah, sorry about that - fixed.